### PR TITLE
codeception EA07BasicinfoCest

### PR DESCRIPTION
**Pull request by hoand-vn**
## 概要(Overview・Refs Issue)
+ PullRequestの目的
  - EA07BasicinfoCest

## 実装に関する補足(Appendix)
+ redefine in file .env: MAILER_URL=null//localhost ==> MAILER_URL=smtp://mailcatcher:1025
+ Change some pages

## Discussion
+ run each case are okie
+ run all cases maybe problem : loose session so back to admin/login page
 
## テスト（Test)
+ CodeCeption local

### CODE REVIEW

File: tests/_support/Page/Admin/TaxManagePage.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function 入力_適用日時($value) {
     }
 
     public function 入力_個別税率設定($value) {
-        $this->tester->selectOption(['id' => 'tax_rule_option_product_tax_rule_0'], $value);
+        $this->tester->checkOption('#tax_rule_option_product_tax_rule_0');
```
Comment (kiy0taka) [inline tests/_support/Page/Admin/TaxManagePage.php]: Unused variable `$value`.

File: tests/acceptance/EA07BasicinfoCest.php [hunk 2]
```diff
@@ -211,15 +211,15 @@ public function basicinfo_税率設定(\AcceptanceTester $I)
...
             ->入力_消費税率(12)
             ->共通税率設定_登録();
 
         $I->see('税率設定情報を保存しました。', TaxManagePage::$登録完了メッセージ);
-        $I->see('12%', $TaxManagePage->一覧_税率(2));
+        $I->see('12%', $TaxManagePage->一覧_税率(1));
 
         // 削除
-        $TaxManagePage->一覧_削除(2);
+        $TaxManagePage->一覧_削除(1);
```
Comment (kiy0taka) [inline tests/acceptance/EA07BasicinfoCest.php]: Can you add assertion for deletion?

Comment (hoand-vn) [reply tests/acceptance/EA07BasicinfoCest.php]: @kiy0taka san, please help us check it again

File: tests/_support/Page/Admin/TaxManagePage.php (hunk 1 above)
Comment (kiy0taka) [reply tests/_support/Page/Admin/TaxManagePage.php]: I think this method needs to be able to change the value of `個別税率`.
`$this->tester->checkOption('#tax_rule_option_product_tax_rule_0')` can only set `個別税率` to `有効`.
`$this->tester->selectOption(['id' => 'tax_rule_option_product_tax_rule_0'], $value)` can set `個別税率` to `有効` or `無効` by `$value`.

File: tests/acceptance/EA07BasicinfoCest.php (hunk 2 above)
Comment (kiy0taka) [reply tests/acceptance/EA07BasicinfoCest.php]: 👍

File: tests/_support/Page/Admin/TaxManagePage.php (hunk 1 above)
Comment (hoand-vn) [reply tests/_support/Page/Admin/TaxManagePage.php]: thank you @kiy0taka san, we  checked again previous solution and it work :(.
so rollback the code

**Comment by kiy0taka** [comment]
Thanks all!

**Pull request merged**